src/main/java/pl/hk/zadanie_31/controller/WeatherDto.java

```diff
@@ -0,0 +1,19 @@
+package pl.hk.zadanie_31.controller;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class WeatherDto {
+
+    private String name;
+    private double temp;
```

Comment:
Przez to, że używasz typy proste, a nie obiektowe to dostajesz błąd, że wartość "null" jest odrzucana przy mapowaniu. Żeby naprawić ten błąd, zmień typy proste na obiektowe (Double)